Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue5108.xaml.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Xaml;
+using System;
+
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.ManualReview)]
+#endif
+#if APP
+	[XamlCompilation(XamlCompilationOptions.Compile)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 5108, "Frame with a shadow on iOS adds a shadow to the contents", PlatformAffected.iOS)]
+	public partial class Issue5108 : TestContentPage
+	{
+		public Issue5108()
+		{
+#if APP
+			InitializeComponent();
+#endif
+		}
+
+		protected override void Init()
+		{
+		}
+
+
+		private void MarginButton_Clicked(object sender, EventArgs e)
```

Comment:
```suggestion
		void MarginButton_Clicked(object sender, EventArgs e)
```

Reply:
No `private` keyword as per the [coding style](https://github.com/xamarin/Xamarin.Forms/#coding-style)